_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -166,8 +179,7 @@ private static function get_audio_enclosure( SimplePie_Item $episode ) {
 			}
 		}
 
-		// Default to empty SimplePie_Enclosure object.
-		return $episode->get_enclosure();
+		return new \WP_Error( 'invalid_audio', 'Podcast audio is an invalid type.' );
```

Comment:
Do we want this to be translatable?

We can probably remove `\` as well, to stay consistent with the rest of the file.

Reply:
You are absolutely right. Sorry I missed it!

Reply:
Fixed.